contributing/code/standards.rst

```diff
@@ -194,6 +194,9 @@ Structure
 
 * Do not use spaces around ``[`` offset accessor and before ``]`` offset accessor.
 
+* Import classes from other namespaces with the ``use`` statement, except for
+  built-in PHP classes.
```

Comment:
I would reword this a bit:

```rst
* Add a ``use`` statement for every class that is not part of the global namespace.
  Prefix class names from the global namespace with a single backslash.
```

Reply:
Your proposal is more correct ... but I think @umulmrum's proposal is easier to understand: "builtin PHP classes" is more clear to me than "class names from the global namespace". But this is just my opinion!

Reply:
The question beyond wording is, do we speak of built-in classes or of classes in the global namespace? Should user-land classes in the global namespace be imported?

Reply:
If there are user-land classes in the global namespace that we use, we also do not add a `use` statement.

Reply:
I'm :+1: for @xabbuh's proposal, it's more correct and people reading this CS guidelines are expert enough to know what the global namespace is.

Reply:
What about a simplified version of @xabbuh's proposal? "Add a use statement for every class that is not part of the global namespace."

Prefixing a class from the global namespace with a backslash is syntactically required when in a namespaced class, so this doesn't need to be written in a style guide.

Reply:
I like @umulmrum's suggestion.